api/src/main/java/uk/gov/dhsc/htbhf/claimant/service/EligibilityService.java

```diff
@@ -34,7 +34,7 @@
      * @param claimant the claimant to check the eligibility for
      * @return the eligibility and entitlement for the claimant
      */
-    public EligibilityAndEntitlement determineEligibilityForNewClaimant(Claimant claimant) {
+    public EligibilityAndEntitlementDecision determineEligibilityAndEntitlementForNewClaimant(Claimant claimant) {
```

Comment:
...this does now make we wonder whether the name of the service (and this method) should be reviewed as the `EligibilityService` is also determining entitlement...sorry... Would this method be something like `decideForNewClaimant` or `evaluateNewClaimant`?

Reply:
`EligibilityAndEntitlementService`?
`.evaluateNewClaimant` & `.evaluateExistingClaimant`

Reply:
Sounds good to me